jx_sqlite/aggs_table.py

```diff
@@ -360,12 +360,21 @@ def _get(row):
                     column_number = len(outer_selects)
 
                     for json_type, sql in details.sql.items():
-                        concat_sql.append("GROUP_CONCAT(QUOTE(DISTINCT(" + sql + ")))")
+                        concat_sql.append("GROUP_CONCAT(DISTINCT(" + sql + "))")
```

Comment:
...plus , if you surround `concat_sql` with `[]`, you can generate JSON, which will be digested nicely with JSON tools on the Python side.